### PR TITLE
Dev jmv

### PR DESCRIPTION
**Pull request by jmvanson**

### CODE REVIEW

File: docs/source/project_exaDEM/NumericalScheme.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ Formula:
 
   f = a.m;
 
-with **f** the sum of forces applied to the particle, **a** the acceleration of the particle, and m its mass.
+with **f** the sum of forces applied to the particle, **a** the acceleration of the particle, and m its mass. NOTE : We are not sure of the validity of this equation. Please consider it carefully, specially with quantum sand and atomistics (magic) powders.
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/NumericalScheme.rst]: remove space " :" -> ":"

File: docs/source/project_exaDEM/Parallelization.rst [hunk 2]
```diff
@@ -89,7 +89,7 @@ This benchmark has been presented in 2023 within the paper :cite:`Carrard_2024`
 
 Description:
 
-Different OpenMP/Mpi con␜gurations (number of cores/threads per ``mpi`` process) have been tested to balance multi-level parallelism. 
+Different OpenMP/Mpi conigurations (number of cores/threads per ``mpi`` process) have been tested to balance multi-level parallelism.
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Parallelization.rst]: configuration

File: docs/source/project_exaDEM/Test_cases.rst [hunk 3]
```diff
@@ -7,10 +7,10 @@ You can explore various basic test cases located in the `example` directory. The
 Example Using Spheres
 ---------------------
 
-Example 1: Rotating drum
+Example 1 : Rotating drum
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 4]
```diff
@@ -34,10 +34,10 @@ A DEM simulation of a rotating drum involves modeling the movement of spherical
 | |ex1npstart|             | |ex1npend|               |
 +--------------------------+--------------------------+
 
-Example 3: Rigid stress
+Example 2 : Rigid stress
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 5]
```diff
@@ -55,10 +55,10 @@ In a DEM simulation under radial stress, particles are exposed to pressure from
 
 .. warning:: This example is currently disabled. 
 
-Example 4: Rigid surface
+Example 3 : Rigid surface
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 6]
```diff
@@ -74,10 +74,10 @@ A DEM simulation involving spherical particles falling onto a rigid surface offe
 | |ex4start|               | |ex4end|                 |
 +--------------------------+--------------------------+
 
-Example 5: Impose Velocity
+Example 4 : Impose Velocity
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 7]
```diff
@@ -74,10 +74,10 @@ A DEM simulation involving spherical particles falling onto a rigid surface offe
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In this DEM simulation, a scenario is simulated where a group of particles with imposed velocity occupies a defined area. As other particles fall into this region, they interact with the moving particles, impacting their trajectories. The simulation provides insights into how moving particles influence the behavior of surrounding particles. Example: ``impose_velocity_hole``, credit: ``lafourcadp``.  
+In this DEM simulation, a scenario is simulated where a group of particles with imposed velocity occupies a defined area. As other particles fall into this region, they interact with the moving particles, impacting their trajectories. The simulation provides insights into how moving driven-particles influence the behavior of surrounding particles. Example: ``impose_velocity_hole``, credit: ``lafourcadp``.
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: driver

File: docs/source/project_exaDEM/Test_cases.rst [hunk 8]
```diff
@@ -101,10 +101,10 @@ In this DEM simulation, a scenario is simulated where a group of particles with
 | |ex5start2|              | |ex5end2|                |
 +--------------------------+--------------------------+
 
-Example 6 : Movable wall
+Example 5 : Movable wall
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 9]
```diff
@@ -120,10 +120,10 @@ In this DEM simulation, a cluster of spherical particles is constrained against
 | |ex6start|               | |ex6end|                 |
 +--------------------------+--------------------------+
 
-Example 7 : Using a STL Mesh
+Example 6 : Using a STL Mesh
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 10]
```diff
@@ -139,7 +139,7 @@ In this DEM simulation, a cluster of spherical particles is  a cluster of partic
 | |ex7start|               | |ex7end|                 |
 +--------------------------+--------------------------+
 
-Example 9 : Particle Generation With RSA Algorithm
+Example 7 : Particle Generation With RSA Algorithm
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 11]
```diff
@@ -159,7 +159,7 @@ In this DEM simulation, a cluster of 287,642 spherical particles has been genera
 | |ex8start|               | |ex8end|                 |
 +--------------------------+--------------------------+
 
-Example 10 : Jet
+Example 8 : Jet
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 12]
```diff
@@ -188,7 +188,7 @@ This example demonstrates the application of a velocity field to spheres based o
 | |ex10startfull|          | |ex10endfull|            |
 +--------------------------+--------------------------+
 
-Example 11 : Mirror Boundary Conditions
+Example 9 : Mirror Boundary Conditions
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 13]
```diff
@@ -212,7 +212,7 @@ This example tests the mirror conditions available in exaNBody. Although these c
 Examples Using Polyhedra
 ------------------------
 
-Example 1: Polyhedra Generation Frequency
+Example 1 : Polyhedra Generation Frequency
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 14]
```diff
@@ -232,10 +232,10 @@ In this example, we simulate the generation of 100 new polyhedra at every 45000
 | |ex1pstart|              | |ex1pend|                |
 +--------------------------+--------------------------+
 
-Example 2: Octahedra in Rotating Drum
+Example 2 : Octahedra in Rotating Drum
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 15]
```diff
@@ -260,7 +260,7 @@ In this DEM simulation, we observe the dynamics of 125 octahedra as they descend
 +--------------------------+--------------------------+
 
 
-Example 3: Hexapods in Ball 
+Example 3 : Hexapods in Ball
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/Test_cases.rst [hunk 16]
```diff
@@ -280,7 +280,7 @@ This DEM simulation example illustrates the gravitational descent of 64 hexapods
 +--------------------------+--------------------------+
 
 
-Example 4: Polyhedra Wth STL Mesh (Box)
+Example 4 : Polyhedra Wth STL Mesh (Box)
```
Comment (rprat-pro) [inline docs/source/project_exaDEM/Test_cases.rst]: " :" ":"

File: docs/source/project_exaDEM/NumericalScheme.rst (hunk 1 above)
Comment (rprat-pro) [reply docs/source/project_exaDEM/NumericalScheme.rst]: euh non, à supprimer

File: docs/source/project_exaDEM/Parallelization.rst (hunk 2 above)
Comment (rprat-pro) [reply docs/source/project_exaDEM/Parallelization.rst:92]: euh non, à supprimer @jmvanson

**Comment by rprat-pro** [comment]
Merci Beaucoup @jmvanson !!!!!
